sycl/doc/MultiTileCardWithLevelZero.md

```diff
@@ -0,0 +1,125 @@
+# Considerations for programming to multi-tile and multi-card under Level-Zero backend
+
+## 1 Devices discovery
+### 1.1 Root-devices
+
+Intel GPUs are represented as SYCL GPU devices, root-devices.
+The discovery of root-devices is best with "sycl-ls" tool, for example:
+	
+```
+$ sycl-ls
+[opencl:0] GPU : Intel(R) OpenCL HD Graphics 3.0 [21.19.19792]
+[opencl:0] CPU : Intel(R) OpenCL 2.1 [2020.11.11.0.03_160000]
+[level_zero:0] GPU : Intel(R) Level-Zero 1.1 [1.1.19792]
+[host:0] HOST: SYCL host platform 1.2 [1.2]
+```
+
+Note that "sycl-ls" shows all devices from all platforms of all SYCL backends that are seen
+by SYCL runtime. Thus in the example above there is CPU (managed by OpenCL backend) and 2!
+GPUs corresponding to the single physical GPU (managed by either OpenCL or Level-Zero backend).
+There are few ways to filter observable root-devices.
+	
+One is using environment variable SYCL_DEVICE_FILTER described in [EnvironmentVariables.md](https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md)
+```
+$ SYCL_DEVICE_FILTER=level_zero sycl-ls
+[level_zero:0] GPU : Intel(R) Level-Zero 1.1 [1.1.19792]
+```
+Another way is to use similar SYCL API described in [FilterSelector.md](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/FilterSelector/FilterSelector.adoc)
+E.g. `filter_selector("level_zero")` will only see Level-Zero operated devices.
+
+If there are multiple GPUs in a system then they will be seen as multiple different root-devices.
+On Linux these would be multiple SYCL root-devices of the same SYCL platform (representing Level-Zero driver).
+On Windows these would appear as root-devices of multiple different SYCL platforms (Level-Zero drivers).
+
+`CreateMultipleRootDevices=N` evironment variable can be used to emulate multiple GPU cards.

```

Comment:
Could you add comment/note that NEOReadDebugKeys=1 variable should be also set to make RT read mentioned debug options?

Reply:
Added.